src/app/loadout/loadout-share/loadout-import.ts

```diff
@@ -96,6 +96,7 @@ function preprocessReceivedLoadout(loadout: Loadout): Loadout {
   loadout.items = loadout.items.map((item) => ({
     ...item,
     id: item.id === '0' ? generateMissingLoadoutItemId() : item.id,
+    hash: Number(item.hash),
```

Comment:
Do we have evidence that the hash is causing problems? DIM-1T95 shows "Item ID is invalid" in the list of recorded errors, which dim-api throws when the item *id* is invalid (not a number sequence)

Reply:
Fair question. I'd seen a bunch of logs in Sentry where the hash was invalid (the error was 'hash must be a number' from https://github.com/DestinyItemManager/dim-api/blob/master/api/db/loadouts-queries.ts#L138. I wouldn't at all be surprised if the ID is also messed up somehow. I don't exactly know where these loadouts are coming from.

Reply:
You're right that there are also errors about the ID being invalid, but I can't see what the ID was.